upp-concept-publishing-provisioner/README.md

```diff
@@ -31,11 +32,26 @@ docker run \
     -e "ENVIRONMENT_TYPE=$ENVIRONMENT_TYPE" \
     -e "VAULT_PASS=$VAULT_PASS" \
     -e "IS_MULTI_REGION=$IS_MULTI_REGION" \
+    -e "AWS_SECONDARY_REGION=$AWS_SECONDARY_REGION" \
     coco/upp-concept-publishing-provisioner:latest /bin/bash provision.sh
 ```
 
 - You can check the progress of the CF stack creation in the AWS console [here](https://eu-west-1.console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks).
 
+## Manual Step
+
+- Unfortunately there are a couple manual steps that cannot be automated if you provision a multi-region stack.
+- First you must go into the aws console for the secondary region. 
+- Navigate to Simple Nofication Service dashboard and select topics
+- Find the newly created topic associated with your environment and copy the ARN to your clipboard
```

Comment:
In Ansible, is it possible to look up the SNS topic's ARN, then apply permissions for that ARN to the SQS queue, then subscribe?